Applet/AndroidSEProviderLib/src/com/android/javacard/seprovider/KMKeyObject.java

```diff
@@ -0,0 +1,19 @@
+package com.android.javacard.seprovider;
+
+public class KMKeyObject {
```

Comment:
Modify the existing TOO_MANY_OPERATIONS test case to create a new key in the loop instead of using the same key again and upload it as a CL.